pkg/operator/k8sutil/configmap.go

```diff
@@ -63,6 +63,10 @@ func GetOperatorSetting(context context.Context, clientset kubernetes.Interface,
 
 func GetValue(data map[string]string, settingName, defaultValue string) string {
 	if settingValue, ok := data[settingName]; ok {
+		// Suppress logging of ROOK_WATCH_FOR_NODE_FAILURE setting
+		if settingName == "ROOK_WATCH_FOR_NODE_FAILURE" {
+			return settingValue
```

Comment:
How about if we just log it as debug in this case? Then if we have other vars we want to suppress in the future we can add them to the comparison.
```suggestion
			logger.Debugf("%s=%q (configmap)", settingName, settingValue)
		} else {
			logger.Infof("%s=%q (configmap)", settingName, settingValue)			
		}
```